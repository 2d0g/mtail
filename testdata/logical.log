foo
foo
bar
bar
quux
