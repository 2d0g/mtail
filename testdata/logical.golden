counter foo 4 2017-10-03T20:14:42Z
